### PR TITLE
Update Change Log

### PR DESCRIPTION
**Pull request by sebbader-sap**
Adds the Change Log updates from the V3.0.2 Draft also into the V3.1_prep branch.

### CODE REVIEW

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc [hunk 1]
```diff
@@ -64,6 +64,43 @@ add DeleteBulkSubmodelDescriptorsById
...
 |===
 
+
+== Changes w.r.t. V3.0.1 to V3.0.2
+
+Major:
+
+* Remove “format: byte” from OpenAPI files, as this annotation enforces base64 encodings while base64url is actually required.
+* Change the values for the ServiceDescription class from enum to a list in the OpenAPI definition for the Part 2 classes.
+
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:74]:
```suggestion
* Remove "Level" parameter from API-operations with Format content "Metadata"
```

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc [hunk 2]
```diff
@@ -64,6 +64,43 @@ add DeleteBulkSubmodelDescriptorsById
...
+|===
+|*Operation Change Old* |*Operation Change New* |*Kind of Change* |*Comment*
+| GetSubmodelElementByPath-Metadata had ‘level’ and/or ‘cursor’ parameters in OpenAPI |‘level’ and/or ‘cursor’ parameters have been removed GetSubmodelElementByPath-Metadata | Remove | Neither ‘level’ nor ‘cursor’ could influence the behavior of the operation, therefore, removing them has no effect on any implementation. 
+|CreateSubmodelReference| New comment in Table 15 clarifies the content of the Location header of the response. | New |
+|===
+
+
+=== Editorial Changes w.r.t. V3.0.1 to V3.0.2
+- Fix links in Clause 12.1
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:86]:
```suggestion
* Fix links in Clause 12.1
```

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc [hunk 3]
```diff
@@ -64,6 +64,43 @@ add DeleteBulkSubmodelDescriptorsById
...
+- ServiceSpecificationProfileEnum: Corrected explanations for Submodel Repository profile entries
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:87]:
```suggestion
* ServiceSpecificationProfileEnum: Corrected explanations for Submodel Repository profile entries
```

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc [hunk 4]
```diff
@@ -64,6 +64,43 @@ add DeleteBulkSubmodelDescriptorsById
...
+- Correct Note 1 in Clause 12.4: "[…] IdshortPaths are base64url-encoded […]" to "[…] IdshortPaths are url-encoded […]"
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:88]:
```suggestion
* Correct Note 1 in Clause 12.4: "[…] IdshortPaths are base64url-encoded […]" to "[…] IdshortPaths are url-encoded […]"
```

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc [hunk 5]
```diff
@@ -64,6 +64,43 @@ add DeleteBulkSubmodelDescriptorsById
...
+- Clause 12.2: Correct example for GetAllAssetAdministrationShellIdsByAssetLink
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:89]:
```suggestion
* Clause 12.2: Correct example for GetAllAssetAdministrationShellIdsByAssetLink
```

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc [hunk 6]
```diff
@@ -64,6 +64,43 @@ add DeleteBulkSubmodelDescriptorsById
 | |Discovery Profile – Read Profile |new | 
 |===
 
+
+== Changes w.r.t. V3.0.1 to V3.0.2
+
+Major:
+
+* Remove “format: byte” from OpenAPI files, as this annotation enforces base64 encodings while base64url is actually required.
+* Change the values for the ServiceDescription class from enum to a list in the OpenAPI definition for the Part 2 classes.
```
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:73]: missing in the table below: which API-operations are affected?

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
one major change was missing it seems to me + table of API-operations was not complete

+ minor editorial issues

File: documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc (hunk 6 above)
Comment (BirgitBoss) [inline documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_ChangeLog.adoc:73]:
```suggestion
* Change the values for the ServiceDescription class from enum to a list of strings in the OpenAPI definition for the Part 2 classes.
```

**Comment by sebbader-sap** [comment]
> one major change was missing it seems to me + table of API-operations was not complete
> 
> * minor editorial issues

Took me some time to understand what you meant. I think I have it now, please check again. Editorial Changes section is now in the "Minor Changes" right at the beginning.

**Review by BirgitBoss — APPROVED** [review]

**Pull request merged**